classes/Language.php

```diff
@@ -882,6 +882,8 @@ public static function checkAndAddLanguage($iso_code, $lang_pack = false, $only_
             }
         }
 
+        self::loadLanguages();
```

Comment:
Can you explain why you add this? I guess there is a good reason but I don't get it.